control-plane/agents/src/bin/core/controller/reconciler/mod.rs

```diff
@@ -99,6 +99,11 @@ trait GarbageCollect {
     async fn disown_orphaned(&mut self, context: &PollContext) -> PollResult;
     /// Disown resources which have questionable existence, for example non reservable replicas.
     async fn disown_invalid(&mut self, context: &PollContext) -> PollResult;
+    /// Reclaim unused capacity - for example an expanded but unused replica, which may
+    /// hapen as part of a failed volume expand operation.
+    async fn reclaim_space(&mut self, _context: &PollContext) -> PollResult {
+        PollResult::Ok(PollerState::Idle)
+    }
```

Comment:
Wonder if this should be in a separate trait, ex: reclamation

```suggestion
    /// Reclaim unused capacity - for example an expanded but unused replica, which may
    /// happen as part of a failed volume expand operation.
    async fn reclaim_space(&mut self, _context: &PollContext) -> PollResult {
        PollResult::Ok(PollerState::Idle)
    }
```

Reply:
This call won't be most likely implemented for anything else(perhaps may be Pool at some point in future)? Pretty much fits well here also. We can separate later also if it tends to become more generic. Let me know...